Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -5424,3 +5455,358 @@ private command EditorFieldCacheSwap pOldRuggedObjectId, pNewRuggedObjectId
    -- controls.
    __EditorFieldCacheFlushExcess
 end EditorFieldCacheSwap
+
+on folding
+   local tLine
+   local tLineText
+   local tHandlerName
+   local tText
+   local tEndBlock
+   local tCounter
+   
+   put word 2 of the clickLine into tLine
+   put line tLine of field "Script" of me into tLineText
+   
+   -- handlers beginning with on, function or command
+   if word 1 of tLineText is among the items of   "on,private,function,command,setProp,getProp,after,before" then
+      if word 1 of tLineText is "private" then
+         put trueWord 3 of tLineText into tHandlername
```

Comment:
I used trueWord because I found one instance in a getProp handler: getProp xyz[abc] 
I did not try token though.